src/measure_theory/function/conditional_expectation/basic.lean

```diff
@@ -2080,6 +2080,42 @@ begin
   rw [hg_eq s hs hμs, set_integral_condexp hm hf hs],
 end
 
+lemma condexp_bot' (f : α → F') :
+  μ[f|⊥] =ᵐ[μ] λ _, (μ set.univ).to_real⁻¹ • ∫ x, f x ∂μ :=
```

Comment:
If I understand correctly, this is a pointwise equality (and not only almost everywhere) unless the measure is zero, right? If it's indeed the case, I think it would be a good idea to have the stronger statement, and deduce this one from it.